corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml

```diff
@@ -8,6 +8,15 @@
   since: "1.3"
   force: true
 
+- name: "Lazy Load Resources"
+  description: "How should Commcare load the the multimedia files in the application."
+  id: "lazy_load_multimedia_files"
+  values: ["true", "false"]
+  value_names: ["Yes", "No"]
+  default: "false"
+  since: "2.5"
```

Comment:
Why is this 2.5? Wasn't this added recently, maybe 2.49?

Reply:
Think it will be released in 2.5, I confirmed it from @shubham1g5

Reply:
it should be 2.50

Reply:
Version updated to 2.50